electionleaflets/settings/base_lambda.py

```
@@ -66,4 +66,6 @@
 CSRF_TRUSTED_ORIGINS = ["https://electionleaflets.org"]
 USE_X_FORWARDED_HOST = True
 
+YNR_API_KEY = os.environ["YNR_API_KEY"]  # Fail if not API key
```

:+1: for failing loudly if we try to deploy and this is not set